src/implementations/truncation.jl

```diff
@@ -32,6 +32,19 @@ Trivial truncation strategy that keeps all values, mostly for testing purposes.
 """
 struct NoTruncation <: TruncationStrategy end
 
+function to_truncationstrategy(trunc::TruncationStrategy)
```

Comment:
It might be reasonable to use the abstract type for this, and simply call this function `TruncationStrategy`.
The main reason not to is if we are expecting to also support truncation types that are not a subtype, but I don't think that is very likely.
Alternatively, I guess this function would be the equivalent of `select_algorithm`, so perhaps `select_truncation` is fitting?

Reply:
`select_truncation` sounds good to me (I'd also be ok with `TruncationStrategy` but `select_truncation` feels a bit more conservative for now, for example it could remain internal for the time being).